kpops/component_handlers/topic/handler.py

```diff
@@ -146,16 +146,14 @@ def __check_partition_count(
         if partition_count == (
             topic_spec.partitions_count or int(broker_config["num.partitions"])
         ):
-            log.info(
-                yellowify(
-                    f"Topic Creation: partition count of topic {topic_name} did not change. Current partitions count {partition_count}. Updating configs."
-                )
+            log.debug(
+                f"Topic Creation: partition count of topic {topic_name} did not change. Current partitions count {partition_count}. Updating configs."
             )
         else:
             log.error(
                 f"Topic Creation: partition count of topic {topic_name} changed! Partitions count of topic {topic_name} is {partition_count}. The given partitions count {topic_spec.partitions_count}."
             )
-            exit(1)
+            exit(1)  # TODO shouldn't we raise instead of exiting?
```

Comment:
@raminqaf @disrupted Any reason why we exit and don't raise? To me this looks strange especially as we don't have any way to handle it

Reply:
I don't know the reason, but I agree that raising an exception would make sense

Reply:
We can also do that outside this PR, I just stumbled upon it

Reply:
Agree we should raise an Exception. Will you create an issue for it please?

Reply:
#101

Reply:
Should I retain the TODOs in the code (and rephrase them a little)?